code/espurna/config/hardware.h

```diff
@@ -825,6 +825,116 @@
     // Only used when RFB_PROVIDER is RCSWITCH
     #define RFB_RX_PIN          4
     #define RFB_TX_PIN          5
+#elif defined(YAGUSMART_TOUCH_HWMOD_1G)
+    // Info
+    #define MANUFACTURER        "YAGUSMART"
+    #define DEVICE              "1_GANG_SWITCH"
```

Comment:
`TOUCH_HWMOD_1G`? or move DEVICE test into the build flag token
-//- the rest of the .h, and also the .ini

The idea here is to have `MANUFACTURER "_" DEVICE` be the same as the build flag

Reply:
I didn't know.